lib/new-admin/graphql/schema.js

```diff
@@ -174,6 +175,13 @@ const typeDefs = gql`
     ip_address: String
   }
 
+  type Coupon {
+    id: ID!
+    code: String!
+    discount: Int!
+    soft_deleted: Boolean
```

Comment:
We don't really use `_` on the schema, should be camel case (in this case should be removed because we don't really need this on the front-end